tensorflow_addons/image/tests/color_ops_test.py

```diff
@@ -50,6 +51,22 @@ def test_equalize_with_PIL():
     np.testing.assert_equal(color_ops.equalize(tf.constant(image)).numpy(), equalized)
 
 
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
```

Comment:
Probably we could add `@pytest.mark.parametrize` to test different bins

Reply:
@bhack the two most common would probably be 256 and 2^16. I'm not sure if it is a good idea to include more as it would slow the tests down. Additionally, since the test matrices are randomly generated there is a chance a low error tolerance would cause intermittent failures.

I think having 1 test to match the current PIL function and another test to match skimage within some tolerance that doesn't cause transient errors would be ideal. Otherwise we may need to adopt skimage equalize implementation which may be a much larger change than the current PR.

Another way to solve the transient error issue is to load a static test image.

Reply:
Having the two most common is ok. How many bins are testing skimage?

Reply:
@bhack Originally only one random bin size was tested. The most recent commit adds `@pytest.mark.parametrize` to test for bin sizes 256 and 2^16.